### ISSUE TITLE
Collect user's email

### ISSUE THREAD

**Issue by NoodlesDev987** [sylveon/discord-ban-appeals#19]
Add option to collect also user's email from Discord oauth. Then staff will be able to reply to the appeal to user's email.

**Comment by jcsumlin** [comment]
I feel like this is maybe unnecessary since the user's name/discriminator is/can be collected. If an admin needed to reach out it would probably be best to do so within the discord ecosystem rather than exposing a user's or admin's personal information.

**Comment by sylveon** [comment]
Yes, one of the selling points of Discord over other services is email and IP privacy.

This site already blurs the line on the latter because nothing stops the server owner from, for example, logging IPs. I think this would work better with #18, so one can add a question asking for email, and any good faith appeal could put in an email. Another reason for not collecting the email from OAuth is that the user might not want to use their main Discord account email, because it could be on an inbox they don't check regularly for example.

Of course, the peak would be for this system to automatically send emails on approved appeals and never directly expose them to the owner/mods, but it would quickly kill the easy to setup and free to host aspect of this project (would need a database, a service to send emails, etc.)

**Comment by jcsumlin** [comment]
@sylveon Ill have to double-check since it's been a while since I've worked on it, but the ReactJS take on your app that I've published ([jcsumlin/discord-ban-appeal](https://github.com/jcsumlin/discord-ban-appeal)) DM's the banned user (or tries to) once the ban appeal has been approved.

**Comment by sylveon** [comment]
Didn't think of that, genius!

**Comment by NoodlesDev987** [comment]
Ok
